forecasting/src/autogluon/forecasting/task/forecasting/dataset.py

```diff
@@ -0,0 +1,195 @@
+from gluonts.dataset.repository.datasets import get_dataset, dataset_recipes
+from gluonts.dataset.field_names import FieldName
+from gluonts.dataset.common import ListDataset
+import pandas as pd
+
+__all__ = ["TimeSeriesDataset", "transform_tabular_to_gluonts_dataset", "gluonts_builtin_datasets"]
+
+
+def gluonts_builtin_datasets():
```

Comment:
these helper functions should be defined in another file, eg. in `utils/dataset_utils.py`